translate/lang/data.py

```diff
@@ -239,6 +239,75 @@
 }
 
 
+scripts = {
+    'devanagari': [
+        'anp',
+        'bho',
+        'brx',
+        'doi',
+        'hi',
+        'kfy',
+        'kok',
+        'mai',
+        'mr',
+        'sa',
+        'sat',
+    ],
+    'bengali': [
+        'bn_BD',
+        'bn_IN',
+    ],
+    'tibetan': [
+        'bo',
+    ],
+    'assamese': [
+        'as',
+    ],
+    'odia': [
+        'or',
+    ],
+    'gujarati': [
+        'gu',
+    ],
+    'khmer': [
+        'km',
+    ],
+    'kannada': [
+        'kn',
+    ],
+    'persian': [
+        'ks',
```

Comment:
This could be one of three possible scripts https://en.wikipedia.org/wiki/Kashmiri_language and there isn't a Persian script, I'm pretty sure it would fall under Arabic.